docs/content/mobile/overview.md

```diff
@@ -11,9 +11,9 @@ Before starting with mobile development, make sure you are familiar with
 
 ## Developing SpatialOS games for Android and iOS
 
-We provide a feature module for the SpatialOS GDK for Unity to allow you to develop games for Android and iOS. This allows you to build [client-workers]({{urlRoot}}/content/glossary#client-worker) for different platforms that can all connect to the same deployment and enables you to create cross-platform games.  All current feature modules are supported and work with mobile devices.
+The SpatialOS GDK for Unity contains a [mobile feature module]({{urlRoot}}/content/modules/core-and-feature-module-overview#mobile-support-module) which enables you to develop games for Android and iOS. All feature modules work with mobile devices.
 
-Mobile support is in [pre-alpha (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages). Currently, you can only connect your mobile client-workers to a local deployment: you **cannot** connect mobile client-workers to a cloud deployment. Please follow our [Roadmap](https://github.com/spatialos/gdk-for-unity/projects/1) to learn more about updates to this in future releases.
+Because your [SpatialOS World](https://docs.improbable.io/reference/latest/shared/glossary#spatialos-world), the canonical source of truth about your game, is distinct from the view of that world that your client-workers visualise, you can create cross-platform multiplayer games with ease.
```

Comment:
this sentence somehow feels out of place, though I understand why you put it here.

Reply:
fixed :)